### ISSUE TITLE
More granularity with Blocklists/Mutelists/Labels

### ISSUE THREAD

**Issue by SpriterGors** [bluesky-social/social-app#5536]
**Is your feature request related to a problem? Please describe.**

Currently I am using several moderation lists and labels to curate my feeds to the way I want. However sometimes I find out that one of my friends have been added to a moderation list/label created by someone else, forcing me to unsubscribe to them in order to get access to their posts.

**Describe the solution you'd like**

Moderation lists and Labels should allow me to toggle off certain users I'm interested in, so that I have a more granular filtering on my feeds.

**Describe alternatives you've considered**

1. Accessing the moderation list link should list the included users and a toggle to override the block/mute. See mockup:
![image](https://github.com/user-attachments/assets/cc267745-dc8f-4e9f-85a5-289e37759e47)
2. Accessing the blocked/muted user should display a toggle to override the hide warning. See mockup:
![image](https://github.com/user-attachments/assets/27f93c84-2976-4f98-aa73-63b2a04625ac)

In both mockups, there would be a text label like "override unblock/unmute" or something in this line. (this means, toggling them ON would make them visible for you). This way I can keep blocking most of the list without affecting the users I am interested in.

**Additional context**

Sometimes when people create moderation lists, they do so in a manual way, and false positives might crop up. But currently, all you can do with them is to either block/mute all or not, and it feels counter-productive to unblock an entire list just because my friend happened to be included wrongly in the list.

This toggle option would only affect me (the user), not changing the base moderation list in any way. This way each person could have a customized filtering option.

**Comment by surfdude29** [comment]
A similar proposal has been requested before in #3186, and there is also significant discussion of this issue in https://github.com/bluesky-social/atproto/discussions/1978.

**Comment by mschwendener** [comment]
> sometimes I find out that one of my friends have been added to a moderation list/label created by someone else, forcing me to 
> unsubscribe to them in order to get access to their posts.

I agree that this should not happen, and especially not without a warning, a) when I subscribe to a block list, b) when someone I follow is added to a block list I'm subsribed to. 

(Just happened to me, I only found out because I thought it strange that I hadn't seen any posts from them in quite some time.)

**Comment by J-J-Chiarella** [comment]
> (Just happened to me, I only found out because I thought it strange that I hadn't seen any posts from them in quite some time.)

Bingo.

Now imagine how many other people would reach the stage you were when you noticed a lack of updates from a certain person—but then lack the know-how (as you have a GitHub account, that puts you in the top one percent right off the bat) to determine the problem.

If a person is not a technically minded sleuth and/or is going based off the assumption of what makes sense, that person would not understand the cause of the problem. I would assume, even after learning the basics of Bluesky, that if I follow Alice Smith and Alice Smith is on Jeff Jones’s blocklist and I subscribe to Jeff Jones’s blocklist, that the program *by default* would toggle some granular exception (a “whitelist”?) for Alice Smith so that I could *continue* to follow Alice Smith. I may miss out on the next Alice Smith–like person, but I can expect that. I would not naturally expect to be automatically prevented from seeing updates to Alice Smith.

The more I read, the more I think that Bluesky is fine with occupying a small niche, not growing in scope but only securing the loyalty of a community of diehards (diehards for a company that will inevitably have to raise revenue with ads—pathetic to worship a company, really). It is too bad, as I want to avoid the troll farms and personal fortunes of Musk and Zuck on their platforms of X and Threads, but Bluesky seems content to not address any complaints except those by the true believers who see nothing wrong with the block list system as is.
 
It is cult-like and creepy. At least this bug report flies under the radar. So many times, this topic in Reddit, other forums, or on GitHub devolves to the point where the true believers act perplexed as to why someone could complain then bully and harangue the person by insinuating that they deserve whatever inconvenience comes their way and must have done something bad to earn something bad in turn.

That, or they gaslight us by saying that no one owes us attention or that Bluesky has so much freedom, even when cases like this prove the difficulty in following multiple people.

My solution would be to eschew block lists entirely and deal with the not infrequent baddies so as to not miss out on something good. It is a highly flawed suggestion, yes, but the Blueksy engineers are not about to come to the rescue.
